packages/avatar/README.md

```diff
@@ -137,6 +145,12 @@ import { Avatar } from '@spectrum-web-components/avatar';
 </sp-tab-panel>
 </sp-tabs>
 
-## Accessibility
+### States
+
+#### Undefined
```

Comment:
Can you describe the appearance and guidance for avatars with no user-set image. Also there should be a disabled state also for Avatar.